notNeededPackages.json

```
@@ -3750,6 +3750,10 @@
             "libraryName": "roads-server",
             "asOfVersion": "1.0.1"
         },
+        "rolling-rate-limiter": {
+            "libraryName": "rolling-rate-limiter",
+            "asOfVersion": "0.2.0"
```

It was actually added on commit https://github.com/peterkhayes/rolling-rate-limiter/commit/d65f18abad30b6ac108422e6f678f881f11a2d3c
```suggestion
            "asOfVersion": "0.2.6"
```

---

done